lib/pf/api.pm

```diff
@@ -23,6 +23,7 @@ use pf::config::cached;
 use pf::ConfigStore::Interface();
 use pf::ConfigStore::Pf();
 use pf::iplog();
+use pf::fingerbank;
```

Comment:
Ok! Thanks for explanation! I was thinking about the export thing too.
I will add it just to make it "standard" but since I usually avoid EXPORT in modules, that shouldn't change anything